configuration/security.md

```diff
@@ -218,20 +296,33 @@ Limit login attempts on ssh port using tcp: this denies connection if an IP addr
 sudo ufw limit ssh/tcp
 ```
 
-Deny access to port 30 from IP address 192.168.2.1
+Deny incoming requests to port 22 from IP address 192.168.2.1
 
 ```bash
-sudo ufw deny from 192.168.2.1 port 30
+sudo ufw deny in proto tcp from 192.168.2.1 to any port 22
 ```
 
+Allow incoming requests to port 22 from IP address 192.168.2.1
+
+```bash
+sudo ufw allow in proto tcp from 192.168.2.1 to any port 22
+```
+
+Limit incoming requests to port 22 from IP address 192.168.2.1
```

Comment:
Would "Limit incoming requests from IP address 192.168.2.1 to port 22 only" make this easier to read?

Reply:
Good point, I'll change that.